.claude/skills/handoff/SKILL.md

```
@@ -0,0 +1,181 @@
+```bash
+python .claude/skills/handoff/scripts/handoff.py
+```
+
+スクリプトが JSON を stdout に出力する。内容:
+- Plans.md の WIP/TODO/blocked タスク
+- 未コミット diff のサマリー（`git diff --stat`）
+- ブランチ名、最近のコミット
+- Decisions セクション
+
+### Step 2: ��話要約の生成
+
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)